DependencyInjection/SonataMediaExtension.php

```diff
@@ -111,6 +111,15 @@ public function load(array $configs, ContainerBuilder $container)
 
         if (isset($bundles['SonataAdminBundle'])) {
             $loader->load(sprintf('%s_admin.xml', $config['db_driver']));
+
+            if ($container->hasParameter('sonata.admin.configuration.security.role_admin')
+                && $container->hasParameter('sonata.admin.configuration.security.role_super_admin')) {
+                $container->getDefinition('sonata.media.security.superadmin_strategy')
+                    ->replaceArgument(2, array(
+                        $container->getParameter('sonata.admin.configuration.security.role_admin'),
+                        $container->getParameter('sonata.admin.configuration.security.role_super_admin'),
+                    ));
+            }
```

Comment:
Did you actually test this? I'm afraid it won't work if you load the admin bundle after you load the media bundle. I think this should be a job for https://symfony.com/doc/3.1/bundles/prepend_extension.html#index-0

Reply:
Good catch! I usually sort my bundles alphabetically, so you may be right. I'll do some tests anche eventually change this strategy.

Reply:
I know the other strategy works, so this isn't a dead end :)